app/services/doc_auth/acuant/pii_from_doc.rb

```diff
@@ -42,6 +43,17 @@ def convert_date(date)
         Time.zone.at(match[:milliseconds].to_f / 1000).utc.to_date.to_s
       end
 
+      def convert_id_type(state_id_type)
```

Comment:
Was not able to get much data from the code on what other types of ID types we are expecting, and also right now we have it be nil, but we can make it be the actual values that are passed in.